recipes/datview/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "datview" %}
+{% set version = "1.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/datview-{{ version }}.tar.gz
+  sha256: cfc6b19ea6c82e98a47bb43da08216bd0c9fde4ca058d523cb17079448d210d0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  entry_points:
+    - datview = datview.main:main
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+    - tk  
+
+  run:
+    - python >={{ python_min }}
+    - h5py
+    - hdf5plugin
+    - pillow
+    - matplotlib
+    - tk        
+
+test:
+  imports:
+    - datview
+  commands:
+    - datview --help  # [win]
+    - echo "Skipping GUI test on Linux/macOS (headless)..."  # [not win]  
```

Comment:
Add this as a comment, not a dummy test, or see https://conda-forge.org/docs/maintainer/knowledge_base/#yum_requirementstxt docs on how you can add some missing packages to enable GUI tests.